peer-monitoring-service/client/src/tests/mod.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-mod mock;
+pub(crate) mod mock;
```

Comment:
Is this file missing? 😄 Also, if we only use it in 1 location, it's probably okay to inline it and avoid having to add a new file and expose it to the entire crate.